### ISSUE TITLE
Corrupted video frames during the conference streaming.

### ISSUE THREAD

**Issue by kapulkin** [ant-media/Ant-Media-Server#3093]
### Short description
Some frames during the videostreaming come broken. That happens periodically with frequency from 1 time in 10 minutes to several times in a minute. There is a screenshot with a broken frame.

![2021-03-08_16-46-13 (2)](https://user-images.githubusercontent.com/800129/112345809-6b899200-8cd6-11eb-920d-c58c3f81a78b.png)

### Environment

* Server operating system and version: Ubuntu 18.04.5 LTS
* Java version: openjdk 11.0.8 2020-07-14
* Ant Media Server version: enterprise-2.2.1-20201029_2042
* Client operating system and version: MacOS Catalina 10.15.7
* Browser name and version: Chrome 89, Firefox 86

### Steps to reproduce
1. Install Ant-Media Enterprise on remote server, for example in Google Cloud. Server should be powerfull enough to process conference with 3 participants. In my case server is run on Google cloud Virtual machine with 8-Core CPU.
2. Enter license key to activate Ant-Media
3. Open LiveApp settings and enable Adaptive Bitrate with two bitrates - high 3000kbps and low 1500kbps for resolution 720p. Set frame rate to 30 fps. Save settings.

![2021-03-08_16-52-21](https://user-images.githubusercontent.com/800129/112350778-75ad8f80-8cda-11eb-95a9-2bfa167e0918.png)

4. Setup live conference from your computer with three participants. To emulate three participants I opened first tab in Chrome browser, second in incognito mode in Chrome browser, third in Firefox browser.
5. Wait for 10 minutes and look with attention to notice, when corrupted frames appear

### Expected behavior

I have a conference with three participants without corrupted frames, freezes are acceptable.

### Actual behavior

Some times frames in one or two streams become corrupted on period 1-2 seconds.
![2021-03-08_16-59-04](https://user-images.githubusercontent.com/800129/112351064-9bd32f80-8cda-11eb-9286-5492003d29fc.png)

The moment the corrupted frame starts matches the moment in log /usr/local/antmedia/log/ant-media-server.log, when the next message occurs

![2021-03-08_16-59-04](https://user-images.githubusercontent.com/800129/112351300-d2a94580-8cda-11eb-91de-c4a7b4d37a83.png)

### Logs

![2021-03-08_16-59-04](https://user-images.githubusercontent.com/800129/112351361-dd63da80-8cda-11eb-8ecb-506112f9d8a0.png)

I suppose that there is a moment occurs, when the internet speed becomes lower suddenly. Ant-media server switches from one bitrate to another and loses a keyframe or several frames, that precedes the broken frame. In terms of h264 protocol may be you should force to send IDR frame or SI frame as a first frame with a new bitrate.

**Comment by kapulkin** [comment]
The error also occurs on v2.3, installation file: ant-media-server-enterprise-2.3.0-20210302_1432.zip

**Comment by mekya** [comment]
Hi @kapulkin ,

It's likely a packet lost issue in the network.

**Comment by kapulkin** [comment]
@mekya Yes, it looks like a packet loss. I read, that WebRTC could request server to send lost packets again.

> I think we can enable some SVC parameters in VP8 transcoding if applicable.

You better understand, which parameters do you mean. Describe them please, and we can discuss.

The same issue has place for h264. If it is more comfortable for you to work with h264, you could do that.

**Comment by mekya** [comment]
@kapulkin ,

Thank you for your comments. Yes you are right about WebRTC could request server to send lost packets again. I can debug that part.

>You better understand, which parameters do you mean. Describe them please, and we can discuss.

VP8 SVC options are better than H264. SVC makes the video playing more robust against the packet loss.
